src/coverlet.console/Logging/ConsoleLogger.cs

```diff
@@ -7,19 +7,22 @@ namespace Coverlet.Console.Logging
     class ConsoleLogger : ILogger
     {
         private static readonly object _sync = new object();
+        public LogLevel Level { get; set; } = LogLevel.Normal;
 
-        public void LogError(string message) => Log(message, ConsoleColor.Red);
+        public void LogError(string message) => Log(LogLevel.Quiet, message, ConsoleColor.Red);
 
         public void LogError(Exception exception) => LogError(exception.ToString());
 
-        public void LogInformation(string message) => Log(message, ForegroundColor);
+        public void LogInformation(string message, bool important = false) => Log(important ? LogLevel.Minimal : LogLevel.Normal, message, ForegroundColor);
```

Comment:
@ViktorHofer I don't understand new `important` flag. If default level is `Normal` and flag can pass `Minimal` or default `Normal` and we check `if (level < Level) return;`  default level won't skip informations(I think it's what you want).
Or your idea is to have a way to log information also in case of `Quiet`?